server/lib/routes/get-verify-email.js

```diff
@@ -5,12 +5,18 @@
 const url = require('url');
 const got = require('got');
 const logger = require('mozlog')('server.get-verify-email');
-const config = require('../configuration');
-const ravenClient = require('../../lib/raven').ravenMiddleware;
 const joi = require('joi');
+const jwt = require('jsonwebtoken');
+
 const validation = require('../validation');
+const config = require('../configuration');
+const ravenClient = require('../raven').ravenMiddleware;
+const getConnection = require('../utils/parse-connection');
```

Comment:
moved these requires around to have npm installed modules first and internal modules after. Easier to digest imo